projects/plugins/jetpack/modules/sharedaddy/sharing-sources.php

```diff
@@ -3041,3 +3081,125 @@ public function display_footer() {
 		endif;
 	}
 }
+
+/**
+ * Mastodon sharing service.
+ */
+class Share_Mastodon extends Sharing_Source {
+	/**
+	 * Service short name.
+	 *
+	 * @var string
+	 */
+	public $shortname = 'mastodon';
+
+	/**
+	 * Service icon font code.
+	 *
+	 * @var string
+	 */
+	public $icon = '\f10a';
+
+	/**
+	 * Constructor.
+	 *
+	 * @param int   $id       Sharing source ID.
+	 * @param array $settings Sharing settings.
+	 */
+	public function __construct( $id, array $settings ) { // phpcs:ignore Generic.CodeAnalysis.UselessOverridingMethod.Found
```

Comment:
Why ignore the sniff instead of removing the unneeded overriding method?

Reply:
Because that's what we do for all the other sources in this file, and I copied and pasted one of the existing services and started from that? :)

I've removed the method entirely in bbba2e508f58d8f8bb13ea0aeca7d65205e2a1dd